node/express/container/Application/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:6.9.3
+FROM node:10
```

Comment:
Should we not move to Node 12 which is current LTS.

Reply:
Yeah, but with that a lot of tests started failing, hence to minimize the changes. Picked this one. Also 10 is also LTS, hence we have a years time to change ths images.